src/dali/in_transaction.py

```diff
@@ -36,6 +36,9 @@ def __init__(
         asset: str,
         exchange: str,
         holder: str,
+        # validated to be a `configuration.Keyword` enum value at runtime
+        # string is used instead of a `Keyword` type to allow for more flexible input
+        # More info: https://github.com/eprbell/dali-rp2/pull/60#discussion_r928178970)
```

Comment:
I think this comment is not needed (and also not relevant to this PR): all parameters are validated at runime, not just this one (in out- and intra-transactions as well). Let's just remove it for now. If there is a need for additional clarification we can add a developer FAQ (in a separate PR).

Reply:
Removed! For me as a new developer to the project, this was confusing (esp since typing is used pretty consistently through the codebase).